frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -439,20 +408,20 @@ export function FieldValuesWidgetInner({
       >
         {isListMode && isLoading ? (
```

Comment:
Maybe we can clean up this ternary? I feel like nested ternaries can get pretty ugly pretty quickly

Reply:
Agreed but not this PR please